.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Push Docker Image
+
+on:
+  push:
+    branches:
+      - main
+      - production
+
+jobs:
+  docker-build-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Docker Metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ secrets.DOCKERHUB_USERNAME }}/api3-dao-dashboard
+          tags: |
+            # set latest tag for production branch
```

Comment:
The tag should be similar to the other images we have in api3 https://hub.docker.com/u/api3

e.g. `api3/dao-dashboard` (remove the api3 prefix) and maybe full sha hash for the commit tag, like we have [here](https://hub.docker.com/r/api3/airnode-deployer-dev/tags). WDYT?

Reply:
Great suggestions- I implemented both in the latest commit, can see them here: https://hub.docker.com/r/dcroote/dao-dashboard/tags